datastream-server-api/src/main/java/com/linkedin/datastream/server/api/transport/DatastreamRecordMetadata.java

```diff
@@ -14,9 +14,10 @@
   private final int _partition;
   private final String _checkpoint;
   private final int _eventIndex;
+  private final int _sourcePartition;
 
   /**
-   * Construct an instance of DatastreamRecordMetadata. Defaults the event index to 0.
+   * Construct an instance of DatastreamRecordMetadata. Defaults the event index to 0 and source partition to -1.
    * @param  checkpoint checkpoint string
    * @param topic Kafka topic name
    * @param partition Kafka topic partition
```

Comment:
Suggest to update param docs & clarify that "partition" is destination Kafka topic partition

Reply:
done